lib/wanda/messaging/event_registry.ex

```diff
@@ -0,0 +1,13 @@
+defmodule Wanda.Messaging.EventsRegistry do
+  @moduledoc """
+  Cloudevents type registry of the internal representations
+  """
+
+  @execution_completed_event "trento.checks.v1.ExecutionCompleted"
+
+  @spec extract_metadata(any) :: {:ok, String.t(), String.t()} | {:error, any()}
+  def extract_metadata(%Wanda.Execution.Result{execution_id: execution_id}),
+    do: {:ok, @execution_completed_event, execution_id}
```

Comment:
I think this should be `{:ok, {@execution_completed_event, execution_id}}` as tagged tuples always hold two elements in my experience

Reply:
Ok, gonna make it like that! Thanks!

Reply:
Changed here https://github.com/trento-project/wanda/pull/11/commits/402a03b6dc8e4ad95ba02f620bcddb5d0c3e9792